apps/backend/src/application/dto/create-application.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { OmitType } from '@nestjs/swagger';
+
+import { Application } from '../entities/application.entity';
+
+export class CreateApplicationDto extends OmitType(Application, [
+  'id',
+  'status',
+  'user',
+  'userId',
+  'applicationPeriod',
+  'createdAt',
+  'updatedAt',
+]) {}
```

Comment:
it's probably better to just list that one property in this case, took me a minute to find which one is it xd. And some validation would be nice